dayrepo/snippets/models.py

```diff
@@ -121,7 +121,7 @@ class Meta(object):
         blank=False,
         null=False,
     )
-    duties_trouble_id = models.ForeignKey(
+    dutiestrouble_id = models.ForeignKey(
```

Comment:
単語としての区別はつけたほうが読みやすいとかんじましたが、いかがでしょ？
```suggestion
    duties_trouble_id = models.ForeignKey(
```

Reply:
カラム名の変更は公明の罠がとても多く怖いですが、対応しました